server/testing/src/main/java/org/apache/james/util/docker/Images.java

```diff
@@ -25,7 +25,7 @@ public interface Images {
     String ELASTICSEARCH_2 = "elasticsearch:2.4.6";
     String ELASTICSEARCH_6 = "docker.elastic.co/elasticsearch/elasticsearch:6.3.2";
     String ELASTICSEARCH_7 = "docker.elastic.co/elasticsearch/elasticsearch:7.10.2";
-    String OPENSEARCH = "opensearchproject/opensearch:2.4.0";
+    String OPENSEARCH = "opensearchproject/opensearch:2.8.0";
```

Comment:
Please confirm if the 2.8.0 driver can run on the 2.4.0 OpenSearch server 

(IE no breaking changes)

Reply:
... 2.8.0 is the opensearch server :) and the driver is 2.6.0, not 2.4.0 :) It seemed compatible but I will double check